src/main.rs

```diff
@@ -105,6 +105,14 @@ fn main() {
                     println!("Must specify a numeric home_id (in base 16)");
                 }
             }
+            "write_config"          => {
+                let state = ozw.get_state();
+                let controllers = state.get_controllers();
+                for (controller, _) in controllers {
+                    controller.write_config();
+                }
```

Comment:
wondering if we could add a write_configs in the ozw object that would call write_config on all controllers automatically ? what do you think ?